resources/Materials/TestSuite/_options.mtlx

```diff
@@ -72,6 +72,7 @@
       <!-- Method for specular environment sampling (only used for HW rendering):
             0 : Prefiltered: Use a radiance IBL texture that has been prefiltered with the BRDF.
             1 : Filtered Importance Sampling: Use FIS to sample the IBL texture according to the BRDF in runtime.
+            2 : Disable environment mapping.
```

Comment:
Nitpick: could we flip the ordering so that "none" comes first and corresponds to "0". Will make more sense if/when we add other environment methods to this list. Could be flipped so default value "FIS" still is 1.

Reply:
I can redo the C++ enum, but added it at the end for backwards compatibility. If your okay I can renumber. The default I can fix up.